src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from '../Button/Button';
+
+const buttonPanel = () => {
```

Comment:
[OPTIONAL]
Would be awesome to use `map` here to remove duplication and DRY out our code